src/profile/Security.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <BasePage>
+    <template v-slot:header>
+      {{ $vuetify.lang.t('$vuetify.profile.index.2sv') }}
+    </template>
+
+    <ProfileProgress :profile="{user: $user, alternates, mfa}"/>
+
+
+    <v-subheader>
+      You’ll be asked for your second method every 30 days. Drag and drop to reorder method preference
+    </v-subheader>
+
+    <v-list three-line>
+      <v-card class="mx-auto">
+        <template v-for="(item, index) in items">
+          <v-divider v-if="item.divider" :key="index" :inset="item.inset"></v-divider>
+
+          <v-list-item v-else :key="item.title">
+            <v-list-item-avatar>
+              <v-img :src="item.avatar"></v-img>
+            </v-list-item-avatar>
+
+            <v-list-item-content>
+              <v-list-item-title v-html="item.title"></v-list-item-title>
+              <v-list-item-subtitle v-html="item.subtitle"></v-list-item-subtitle>
+            </v-list-item-content>
+
+            <v-btn v-if="item.url" :href="item.url" class="text-capitalize" color="primary">{{ item.btnLabel }}</v-btn>
+          </v-list-item>
+        </template>
+      </v-card>
+    </v-list>
+  </BasePage>
+</template>
+
+<script>
+import ProfileProgress from './ProfileProgress'
+import { recoveryMethods, retrieve as retrieveMethods } from '@/global/recoveryMethods'
+import { mfa, retrieve as retrieveMfa } from '@/global/mfa'
+
+export default {
+  components: {
+    ProfileProgress
+  },
+  data: () => ({
+    alternates: recoveryMethods.alternates,
+    mfa,
+  }),
+  computed: {
+    items: (vm) => [
+      // Todo add these to locales
+      {
+        avatar: require('@/assets/authy-logo.jpg'),
```

Comment:
Whether a particular IDP recommends Authy or Google Authenticator is controlled by the `VUE_APP_TOTP_APP_RECOMMENDATION` env. var., so you'll probably need some logic here to adjust which logo is shown based on that.